istio.deps

```diff
@@ -4,7 +4,7 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "524cfb2be117f343fd25c609bdd622eca7c2b4f5"
+    "lastStableSHA": "e75aa441f21df5f13e26f8517848616f10ebcec4"
```

Comment:
You should not use the commit from the master branch of istio/proxy. Could you change to use the latest commit from the release-1.5 branch of istio/proxy?

Reply:
Oops, looked at the wrong tab. Fixed now. Thanks!